### ISSUE TITLE
the code sample does not work out of the box 

### ISSUE THREAD

**Issue by abhijitv** [MicrosoftDocs/xamarin-docs#1634]
there are compile errors 

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 3ab6098a-39c7-151f-124b-f88aa62c4659
* Version Independent ID: dc27c1e4-ff12-c1af-646a-c2617def0bcd
* Content: [Xamarin.Forms Local Databases - Xamarin](https://docs.microsoft.com/en-us/xamarin/xamarin-forms/app-fundamentals/databases#feedback)
* Content Source: [docs/xamarin-forms/app-fundamentals/databases.md](https://github.com/MicrosoftDocs/xamarin-docs/blob/live/docs/xamarin-forms/app-fundamentals/databases.md)
* Product: **xamarin**
* Technology: **xamarin-forms**
* GitHub Login: @davidbritch
* Microsoft Alias: **dabritch**

**Comment by davidbritch** [comment]
The sample project, linked to from the doc, builds without any errors.

**Comment by abhijitv** [comment]
let me try again.it showed too many errors the last time i tried 
    On Wednesday, 24 April 2019, 3:58:06 pm GMT+8, David Britch <notifications@github.com> wrote:  
 
 
The sample project, linked to from the doc, builds without any errors.

**Comment by abhijitv** [comment]
yes today i checked its building fine. thank you very much for following up! can you remove my comment please ?
    On Wednesday, 24 April 2019, 3:58:06 pm GMT+8, David Britch <notifications@github.com> wrote:  
 
 
The sample project, linked to from the doc, builds without any errors.
